symengine/CMakeLists.txt

```diff
@@ -185,6 +185,11 @@ if (BUILD_SHARED_LIBS)
         SOVERSION ${SYMENGINE_MAJOR_VERSION}.${SYMENGINE_MINOR_VERSION}
     )
 endif()
+
+if (CMAKE_CXX_COMPILER_ID MATCHES "GNU|Clang")
+    set_target_properties(symengine PROPERTIES COMPILE_FLAGS "-Wconversion -Wno-sign-conversion")
```

Comment:
This warning is enabled by default and in travis-ci all warnings are converted to errors